### ISSUE TITLE
AssignNodesToEpsFromHashesImpl Failed to find kernel def hash (14280390279553192696) in kernel registries for Einsum(12) node with name 'Einsum_382'.

### ISSUE THREAD

**Issue by piekey1994** [microsoft/onnxruntime#10117]
**Describe the bug**
I exported the onnx file with pytorch 1.10 on PC. Then I convert the file to ORT format through onnxruntime 1.10. This is the config file:
ai.onnx;1;LayerNormalization
ai.onnx;9;ConstantOfShape,Where
ai.onnx;12;Einsum
ai.onnx;13;ArgMax,Cast,Concat,Gather,Log,LogSoftmax,Pad,Shape,Slice,Softmax,Transpose,Unsqueeze
ai.onnx;14;Add,Div,Mul,Relu,Reshape,Trilu
com.microsoft;1;DynamicQuantizeMatMul

On x86pc, I can load ort file and run it normally with the x86java version of onnxruntime.

But error encountered when loading ort model in Android.
ai.onnxruntime.OrtException: Error code - ORT_FAIL - message: inference_session.cc:1194 AssignNodesToEpsFromHashesImpl Failed to find kernel def hash (14280390279553192696) in kernel registries for Einsum(12) node with name 'Einsum_382'.

**System information**
OS Platform and Distribution: windows 64bit
android studio version : android studio 4.1.1
used android device : emulator(android api level 28 x86)
ONNX Runtime version: 'com.microsoft.onnxruntime:onnxruntime-mobile:1.10.0'

**Comment by skottmckay** [comment]
The pre-built mobile package has a limited set of supported operators and opsets in order to keep the binary size small. This set is based on the operators used by the most popular models. Unfortunately Einsum is not in that set.

https://onnxruntime.ai/docs/reference/operators/mobile_package_op_type_support_1.10.html

You would need to do a custom build to create an onnxruntime android package that includes the Einsum kernel.

**Comment by gwang-msft** [comment]
If the binary size of ONNX Runtime for Android is of no concern, you may also try to use the full build of ONNX Runtime Mobile [here](https://github.com/gwang-msft/unofficial_onnxruntime_mobile_releases)
